File: 2trabajoConstructores/c#/claseNormal/AnimalSinParametros.cs
public class AnimalSinParametros
{
    public string Nombre { get; set; }
    public string Sonido { get; set; }

    public AnimalSinParametros()
    {
        Nombre = "Animal genérico";
        Sonido = "Sonido desconocido";
    }

    public void HacerSonido()
    {
        Console.WriteLine($"{Nombre} hace: {Sonido}");
    }
}

File: 2trabajoConstructores/go/claseNormal/animal_sin_parametros.go
package main;

import "fmt"

type AnimalSinParametros struct {
	Nombre string
	Sonido string
}

func NuevoAnimalSinParametros() AnimalSinParametros {
	return AnimalSinParametros{
		Nombre: "Desconocido",
		Sonido: "Ninguno",
	}
}

func (a AnimalSinParametros) HacerSonido() {
	fmt.Printf("%s hace: %s\n", a.Nombre, a.Sonido)
}
